interfaces/builtin/docker_support.go

```diff
@@ -536,6 +537,24 @@ const dockerSupportPrivilegedSecComp = `
 @unrestricted
 `
 
+const dockerSupportConnectedPlugUdev = `
+# This file contains udev rules for Docker Daemon.
+#
+# Do not edit this file, it will be overwritten on updates
+
+KERNEL=="loop-control", TAG+="%[1]s"
+KERNEL=="loop[0-9]*", TAG+="%[1]s"
+KERNEL=="device-mapper", SYMLINK+="mapper/control", TAG+="%[1]s"
+
+# When devicemapper is specified as backend storage driver(default is aufs on ubuntu),
+# device nodes are created in /dev/mapper/docker-* and linked to /dev/dm-*
+# The corresponding apparmor rule is
+# /dev/mapper/docker* rw,
+#
+# dmsetup tool is required to apply the following udev rule. However it looks like we don't have dmsetup pre-installed on core.
+KERNEL=="dm-[0-9]*", PROGRAM="/sbin/dmsetup info -c --noopencount --noheadings -o name -j %%M -m %%m", NAME="%%k", SYMLINK+="mapper/%%c", TAG+="%[1]s"
```

Comment:
Yes, docker supports to add a rule to the cgroup allowed devices list.
I'll drop udev rules for docker-support interface in this case and document the reason.
Thanks for the mention.